src/create-model.jl

```diff
@@ -905,6 +918,22 @@ function create_model(
             accumulate_capacity_simple[y ∈ Y, a ∈ Ai[y]],
             sum(assets_investment[yi, a] for yi in Yi[a] if starting_year[(y, a)] ≤ yi ≤ y)
         )
+
+        @expression(
+            model,
+            accumulate_capacity_compact[
+                a in A,
+                y in Y,
+                v in V;
+                starting_year_compact[(y, a)] ≤ v ≤ y,
+            ],
+            graph[a].initial_units[y] + # need extra v index
+            (haskey(assets_investment, (v, a)) ? assets_investment[v, a] : 0) - sum(
+                assets_decommission_compact[a, i, v] for
+                i in Y if starting_year_compact[(y, a)] < i ≤ y &&
+                haskey(assets_decommission_compact, (a, i, v))
+            )
+        )
```

Comment:
If you create a new set of investable assets with a compact method, this equation simplifies a lot. Rebase to the latest main, and look at the PR that solves #786 for a reference